docker/almalinux-image.docker

```diff
@@ -20,17 +19,21 @@ RUN yum -y install \
    sudo \
    python3-pip
 
-RUN until curl -S -s --retry 9 --retry-connrefused --retry-delay 1 http://mirror.centos.org/centos/7/os/x86_64/Packages/ant-junit-1.9.4-2.el7.noarch.rpm -o ant-junit-1.9.4-2.el7.noarch.rpm ; do echo "curl failed… trying again… " ; done
-
-RUN rpm -i --nodeps ant-junit-1.9.4-2.el7.noarch.rpm
-
 RUN pip3 install --upgrade pip
 
 # install Sphinx to generate docs
 RUN pip3 install \
    Sphinx \
    sphinx_rtd_theme
 
+# install ant
+RUN mkdir /usr/share/ant && \
```

Comment:
I would

1) move ENVs to the top of this docker file, I do not like when ENV's are all over the place
2) I would make environment property out of Ant version too (ANT_VERSION=1.10.12) and I would use it in wget and tar commands. This way, you may test your image with different Ant replacing it at one place or injecting it from outside

https://www.baeldung.com/ops/docker-container-environment-variables